src/components/scrollable/style.scss

```diff
@@ -20,6 +20,6 @@ $react-custom-scrollbars: '.ScrollbarsCustom';
     &-Scroller,
     &-Content,
     &-Wrapper {
-        height: 100%;
+        //height: 100%;
```

Comment:
Maybe this update not belong to this PR~ I think this is trying to fix the #588 ?

Reply:
I think this PR is only for the language pack~

Reply:
> 

Yes. I mixed several commits with PR.
I'm not familiar with working with you.

It is difficult to reverse the commit, so after this PR, I will request a more elaborate PR.

Reply:
> belong to

Yes. Commit and PR were confused.
#588 Please take care of @wewoor.

Reply:
> > 에 속하는
> 
> 예. 커밋과 PR이 혼동되었습니다. [# 588는](https://github.com/DTStack/molecule/issues/588) 알아서하십시오 @wewoor .

Can i revert?